remo/base/static/base/css/mapbox.css

```diff
@@ -465,486 +465,486 @@
+    width:175px;
+    border-radius:0 2px 2px 0;
+    border-left:0;
+    opacity: 1;
+}
+
+.leaflet-bar .leaflet-control-mapbox-geocoder-toggle,
+.leaflet-bar .leaflet-control-mapbox-geocoder-toggle:hover {
+    border-bottom: none;
+}
+
+.leaflet-control-mapbox-geocoder-toggle {
+    border-radius:2px;
+    -webkit-border-radius:2px;
+}
+
+.leaflet-control-mapbox-geocoder.active,
+.leaflet-control-mapbox-geocoder.active .leaflet-control-mapbox-geocoder-toggle {
+    border-top-right-radius: 0px;
+    border-bottom-right-radius: 0px;
+    -webkit-border-top-right-radius: 0px;
+    -webkit-border-bottom-right-radius: 0px;
+}
+
+.leaflet-control-mapbox-geocoder .leaflet-control-mapbox-geocoder-form input {
+    border:0;
+    width:175px;
+    padding-left:10px;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    height:26px;
+
+    outline: none;
+}
+
+.leaflet-control-mapbox-geocoder .mapbox-geocoder-toggle {
+    position:absolute;
+    top:0;
+    left:0;
+}
+
+.leaflet-control-mapbox-geocoder input:focus {
+    border-color: #888;
+}
+
+.leaflet-control-mapbox-geocoder-results {
+    position:absolute;
+    left:26px;
+    top:26px;
+}
+
+.leaflet-control-mapbox-geocoder-results a,
+.leaflet-control-mapbox-geocoder-results span,
+.leaflet-control-mapbox-geocoder-results a:hover {
+    width:176px;
+    padding:0 10px;
+    text-overflow:ellipsis;
+    border:1px solid #ccc;
+    border-top:0;
+    white-space:nowrap;
+    display:block;
+    background:#fff;
+    line-height:26px;
+    overflow:hidden;
+    text-align:center;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.leaflet-control-mapbox-geocoder-results a:hover {
+    background:#eee;
+}
+
+.leaflet-control-mapbox-geocoder-results a:last-child,
+.leaflet-control-mapbox-geocoder-results span:last-child {
+    border-bottom:1px solid #ccc;
+}
+
+/* Dark Theme */
+
+.leaflet-container.dark .leaflet-bar {
+    border-color: #000;
+}
+
+.leaflet-container.dark .leaflet-bar a,
+.leaflet-container.dark .leaflet-control .mapbox-button {
+    background-color: #404040;
+    color: #fff;
+    border-color: #000;
+}
+
+.leaflet-container.dark .leaflet-bar a:hover,
+.leaflet-container.dark .leaflet-control .mapbox-button:hover {
+    background-color: #565656;
+}
+
+.leaflet-container.dark .leaflet-bar a:active,
+.leaflet-container.dark .leaflet-control .mapbox-button:active {
+    background-color: #6B6B6B;
+}
+
+.leaflet-container.dark .leaflet-bar a.leaflet-disabled,
+.leaflet-container.dark .leaflet-control .mapbox-button.disabled {
+    background-color: #252525;
+    color: #474747;
+}
+
+.leaflet-container.dark .leaflet-control-mapbox-geocoder input[type=text] {
+    background-color: #565656;
+    border-color: #000;
+    color: #fff;
+}
+
+.leaflet-container.dark .leaflet-control-mapbox-geocoder input:focus {
+    border-color: #ccc;
+}

```

Comment:
It seems that your editor added some special character next to each line. I'll try to fix it and amend your commit.